src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = days * 40;
+
+  if (days < 3) {
+    return rent;
+  }
+
+  if (days >= 3 && days < 7) {
+    return rent - 20;
+  }
+
+  return rent - 50;
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, I don't know what 50, 20, 7 and 3 are